### PR TITLE
Add a remote boolean for Distribution

### PR DESCRIPTION
**Pull request by hennevogel**
In preparation to move remote Distributions from memcache to a database cache so we can use ActiveRecord for them.

We should not merge this together with the rest of the refactoring (coming later).

### CODE REVIEW

**Review by saraycp — APPROVED** [review]

File: src/api/db/data/20210520160000_set_default_for_distro_remote.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class SetDefaultForDistroRemote < ActiveRecord::Migration[6.0]
```
Comment (krauselukas) [inline src/api/db/data/20210520160000_set_default_for_distro_remote.rb:1]: @hennevogel why is this necessary? you set the default in the db migration through `change_column_default` already and the field is new

Comment (hennevogel) [reply src/api/db/data/20210520160000_set_default_for_distro_remote.rb:1]: https://github.com/ankane/strong_migrations#adding-a-column-with-a-default-value

File: src/api/db/data/20210520160000_set_default_for_distro_remote.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  def up
```
Comment (krauselukas) [inline src/api/db/data/20210520160000_set_default_for_distro_remote.rb:2]: @hennevogel why is this necessary? you set the default in the db migration through change_column_default already and the column is new

**Review by krauselukas — CHANGES_REQUESTED** [review]

File: src/api/db/data/20210520160000_set_default_for_distro_remote.rb (hunk 2 above)
Comment (hennevogel) [reply src/api/db/data/20210520160000_set_default_for_distro_remote.rb:2]: https://github.com/ankane/strong_migrations#adding-a-column-with-a-default-value
